inference-engine/tests/functional/plugin/cpu/single_layer_tests/pad.cpp

```diff
@@ -4,57 +4,87 @@
 
 #include <shared_test_classes/single_layer/pad.hpp>
 #include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
 using namespace InferenceEngine;
 using namespace CPUTestUtils;
+using namespace ov;
+using namespace test;
 
 namespace CPULayerTestsDefinitions {
 
-typedef std::tuple<
-        LayerTestsDefinitions::padLayerTestParamsSet,
+using PadInputShapes = std::pair<std::vector<ov::PartialShape>, std::vector<ov::Shape>>;
```

Comment:
The same comments about overhaul test instance changes. Please refer to https://github.com/openvinotoolkit/openvino/pull/8375 comments regarding tests.

Reply:
Of course, done. Thanks